segmentation_models_pytorch/encoders/__init__.py

```diff
@@ -38,17 +43,38 @@
 encoders.update(mobilenet_encoders)
 encoders.update(xception_encoders)
 encoders.update(timm_efficientnet_encoders)
-encoders.update(timm_resnest_encoders)
-encoders.update(timm_res2net_encoders)
-encoders.update(timm_regnet_encoders)
 encoders.update(timm_sknet_encoders)
-encoders.update(timm_mobilenetv3_encoders)
-encoders.update(timm_gernet_encoders)
 encoders.update(mix_transformer_encoders)
 encoders.update(mobileone_encoders)
 
 
+def is_equivalent_to_timm_universal(name):
+    patterns = [
+        "timm-regnet",
+        "timm-res2",
+        "timm-resnest",
+        "timm-mobilenetv3",
+        "timm-gernet",
+    ]
+    for pattern in patterns:
+        if name.startswith(pattern):
+            return True
+    return False
+
+
 def get_encoder(name, in_channels=3, depth=5, weights=None, output_stride=32, **kwargs):
+    if name.startswith("timm-"):
+        warnings.warn(
+            "`timm-` encoders are deprecated and will be removed in the future. "
+            "Please use `tu-` equivalent encoders instead (see 'Timm encoders' section in the documentation)."
```

Comment:
```suggestion
            "Please use `tu-` equivalent encoders instead (see 'Timm encoders' section in the documentation).",
            DeprecationWarning
```
Instead of the default warning class